domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/statementofmeans/Child.java

```diff
@@ -22,8 +23,10 @@
     private final AgeGroupType ageGroupType;
 
     @NotNull
+    @Min(0)
```

Comment:
i prefer if the smaller annotations go on top, i find it easier to read but up to you 👍

Reply:
ok